recipe/meta.yaml

```diff
@@ -14,9 +14,14 @@ source:
     - CMakeLists.patch  # [win and vc<14]
 
 build:
-  number: 2
+  number: 3
+  # Missing CDT packages on s390x and ppc64le
+  skip: True  # [win or s390x or ppc64le]
```

Comment:
Can you elaborate on why `win` is skipped?

Reply:
Mainly because it has never been integrated (in our repo), but you're right, no shortcuts...

Reply:
Enabled Windows build.